### PR TITLE
Reverts mistakes

### PR DESCRIPTION
**Pull request by skhomuti**
- fix receipt error

### CODE REVIEW

File: reverted-tx-watcher/src/agent-reverted-tx.ts [hunk 1]
```diff
@@ -67,32 +67,35 @@ async function handleRevertedTx(
...
+  const suitableAddresses = addresses.filter(([, address]) =>
+    eventAddresses.includes(address.toLowerCase())
+  );
+
+  const receipt = await ethersProvider.getTransactionReceipt(
+    txEvent.transaction.hash
+  );
+  if (!receipt) return;
+  if (receipt.status !== 0) return;
+
```
Comment (vgorkavenko) [inline reverted-tx-watcher/src/agent-reverted-tx.ts:82]:
```suggestion
  const receipt = await ethersProvider.getTransactionReceipt(
    txEvent.transaction.hash
  );
  if (!receipt) return;
  if (receipt.status !== 0) return;

  const eventAddresses = Object.keys(txEvent.addresses).map((a) =>
    a.toLowerCase()
  );
  const suitableAddresses = addresses.filter(([, address]) =>
    eventAddresses.includes(address.toLowerCase())
  );

```

Comment (skhomuti) [reply reverted-tx-watcher/src/agent-reverted-tx.ts:82]: nope, it doesn't make sense to make a receipt request if we haven't suitable addresses

Comment (skhomuti) [reply reverted-tx-watcher/src/agent-reverted-tx.ts:82]: aaah I forget to add exit condition -_-

Comment (skhomuti) [reply reverted-tx-watcher/src/agent-reverted-tx.ts]: fixed

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**